### DISCUSSION TITLE
Aerospace dies when screen is locked

### DISCUSSION THREAD

**Discussion by marcelofern** [nikitabobko/AeroSpace#690] [questions-and-answers]
Hey folks,

I've just noticed that when the computer screen locks due to lack of activity,
AeroSpace dies and I need to restart it (and put all windows in their specific
workspaces).

Is there a way to prevent that?

Thanks,
Marcelo.

**Comment by altitude** [comment]
Same here, have you found a solution eventually @marcelofern?

**Comment by nikitabobko** [comment]
Please check if there any crash reports in Console.app
